global.json

```diff
@@ -1,6 +1,9 @@
 {
-  "msbuild-sdks": {
-    "MSBuild.Sdk.Extras": "3.0.23",
-    "Microsoft.Build.NoTargets": "2.0.1"
-  }
-}
\ No newline at end of file
+    "msbuild-sdks":  {
+                         "MSBuild.Sdk.Extras":  "3.0.23",
+                         "Microsoft.Build.NoTargets":  "2.0.1"
+                     },
+    "sdk":  {
+                "version":  "6.0.100-preview.3.21201.23"
```

Comment:
Remove

Reply:
Yeah something keeps adding this automatically. I always have to remove it.